.github/workflows/python.yaml

```diff
@@ -35,6 +35,12 @@ jobs:
     - name: Run unit tests with pytest and check coverage
       run: ./scripts/coverage.sh
 
+    - name: Upload code coverage
```

Comment:
I would like to get back to 100% coverage soon which would make this pointless right?

Reply:
Kind of... On the other hand, it will be truly 100% coverage, not just an image.

Reply:
What do you mean just an image? I guess what I am saying is that I don't see a point in the badge if we have 100% coverage? Or is it just so that people know that it is tested?

Reply:
Yeah, it's mostly so that people know. I assume that it will help to make a decision about using this package in prod.

Reply:
Okay, sounds good to me.